ignition/service/resourcedriver.py

```diff
@@ -288,6 +292,10 @@ def execute_lifecycle(self, lifecycle_name, driver_files, system_properties, res
             execute_response = self.handler.execute_lifecycle(lifecycle_name, driver_files_tree, PropValueMap(system_properties), PropValueMap(resource_properties), PropValueMap(request_properties), associated_topology, deployment_location)
             if self.async_enabled is True:
                 self.__async_lifecycle_execution_completion(execute_response.request_id, deployment_location, tenant_id)
+            else:
+                if(isinstance(execute_response, LifecycleExecution)):
```

Comment:
Why execute_response object is instance of class LifecycleExecution instead of LifecycleExecutionResponse as this is the lifecycle execution response object.

Reply:
@lokanalla  This object should be an instance of LifecycleExecution as Netconf Driver is sending the Lifecycle execution status immediately with status, failure_details etc.
Please refer to the related PR. https://github.com/IBM/netconf-driver/pull/10

Also you can refer to this link https://github.com/IBM/netconf-driver/blob/9a676f0c3a2e36191bc7f2e1873aa8a406b000e2/netconfdriver/service/resourcedriver.py#L63

Reply:
@suryadipnag , The netconf driver used to send LifecycleExecutionResponse in success case. Any reason to change it to LifecycleExecution object in success case as well?

Reply:
@lokanalla  Previously in Netconf driver we were sending the response as LifecycleExecutionResponse because it was considered as asynchronous job but now we have made the Netconf driver to synchronous job so that is why we are using LifecycleExecution to send the job status, failure reason (if any) immediately.